.github/PULL_REQUEST_TEMPLATE/bug-pull-request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug
+about: Provide specifics about a PR for a bug fix
+title: 'PR/Bug: ____(provide a descriptive name)'
+labels: bug, p1
+assignees: ''
+
+---
+<!-- IMPORTANT: Please do not create a PR without creating an issue first. -->
+
+<!-- Fields in **bold** are REQUIRED, fields in *italics* are OPTIONAL. -->
+
+**Description**
+<!-- A description of how this PR resolved the specified bug-->
+
+<!-- Add any linked issue(s) -->
+Fixes #
+
+*Screenshots or additional context*
+<!-- Add any other context about the problem here and/or screenshots to help explain the problem. -->
+
+*Testing (if applicable)*
+<!-- Explain how you tested this bug fix so that others can replicate it. -->
+<!-- Example: The exact commands you ran and their output. -->
+
+**Bump version information**
+Bump version: v0.xx.x -> v0.xx.x
+
+- [ ] README.md, [installation instructions](../../README.md#installation-instructions)
+- [ ] README.md, [changelog](../../README.md#changelog)
```

Comment:
This isn't really a bump but a changelog update. Should we put it separate it from Bump version?